src/bokeh/plotting/_figure.py

```diff
@@ -784,21 +795,35 @@ class BaseFigureOptions(Options):
     and added.
     """)
 
+RangeLike = Either(
+    Instance(Range),
+    Either(
+        Tuple(Float, Float),
+        Tuple(Datetime, Datetime),
+        Tuple(TimeDelta, TimeDelta),
+    ),
+    Seq(String),
+    Object("pandas.Series"),
+    Object("pandas.core.groupby.GroupBy"),
+)
+
+AxisType = Nullable(Either(Auto, Enum("linear", "log", "datetime", "mercator")))
+
 class FigureOptions(BaseFigureOptions):
 
-    x_range = Any(help="""
+    x_range = Use(RangeLike, default=InstanceDefault(DataRange1d), help="""
```

Comment:
What I wanted here is:
```py
x_range = RangeLike(default=InstanceDefault(DataRange1d), help="""...""")
```
i.e. making properties callable with `__call__`, which would clone the property and allow to override `default` and `help`. However, that would require to implement cloning for all properties, which is a lot of work. So for now I settled on this approach, i.e. `Use(SomeProperty, default=..., help=...)`, which is very similar to `Required()`, just the value for default is different (`Intrinsic` vs. `Undefined`).

Reply:
The name `Use` does not really speak to me about what the purpose is. If it Is just something temporary you plan to get rid of soon-ish, you could call it `_Use` instead.

Reply:
Implemented cloning and dropped `Use`.